### PR TITLE
Added aliases for property editors

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

## Product & version (if relevant)

CMS v14 and v15

## Deadline (if relevant)

Anytime

### CODE REVIEW

File: 15/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/dropdown/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Dropdown
 
-`Alias: Umbraco.DropDown.Flexible`
+`Schema alias: Umbraco.DropDown.Flexible`
+
+`UI Alias: `Umb.PropertyEditorUi.Dropdown`
```
Comment (eshanrnh) [inline 15/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/dropdown/README.md:5]:
```suggestion
`UI Alias: Umb.PropertyEditorUi.Dropdown`
```

**Comment by sofietoft** [comment]
Looks great 💪 
Just one minor thing:
Alias is with a capital A for UI Alias but not for Schema alias 🙈 
Not sure if it should be capital or not, but I think it should at least be consistent!

**Comment by eshanrnh** [comment]
Good point, @sofietoft I've changed the Schema alias casing to be consistent with the UI alias casing 😀

**Review by sofietoft — APPROVED** [review]
Perfect! Looks great 😄

**Pull request merged**